### ISSUE TITLE
Doesn't support '+' in file name

### ISSUE THREAD

**Issue by george-semenov** [fakefs/fakefs#317]
Works:
```ruby
file = 'foo+bar'
File.open(file, 'w+') {|f| f.write('test')}
File.open(file) {|f| puts f.read}
```

Fails with Errno::ENOENT: No such file or directory - foo+bar:
```ruby
require 'fakefs' 

FakeFS do
  file = 'foo+bar'
  File.open(file, 'w+') {|f| f.write('test')}
  File.open(file) {|f| puts f.read} 
end
```

**Comment by sometimesfood** [comment]
@george-semenov: This seems to be related to #133 and possibly #264.

**Comment by sometimesfood** [comment]
This is fixed in fakefs 0.9. Commit 023fca7fe92d5ccfd48c9044b36493450e0151b5 seems to have helped.

@george-semenov: Can you please confirm and close this issue?
